ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -78,6 +79,16 @@ class CommandBarButton: UIButton {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        guard let accentImageView = accentImageView, let imageView = imageView else {
+            return
+        }
+
+        accentImageView.frame = imageView.frame
```

Comment:
why we doing frame calculation instead of autolayout?

Reply:
No specific reason other than the internal example I was looking at used frame calculation. Updated the `accentImageView` to constraint itself to the `UIButton.imageView` `centerXAnchor` and `centerYAnchor`. Confirmed that the image is positioned correctly on size class changes, rotations, and if a title shown in the button.